zabbix-monitoring-cluster/nested/clusterNodes.json

```diff
@@ -0,0 +1,206 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourcePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Prefix string for all resources."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin user name for cluster VMs."
+      },
+      "defaultValue": "monicake"
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password for cluster VMs"
+      }
+    },
+    "vmCount": {
```

Comment:
looks like you only make 1 storage account. We recommend putting no more than 20-40 VM OS disks in a single storage account (20 is better; 40 is pushing it); reference here: https://azure.microsoft.com/en-us/documentation/articles/virtual-machine-scale-sets-design-overview/